lib/util.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const uv = process.binding('uv');
+const Debug = require('vm').runInDebugContext('Debug');
```

Comment:
@domenic: Should we add support for `Weak{Set,Map}` inspection through `Debug` as a follow-on to this PR?